styleguide/derek/view-templates/resource-center/_partials/resource-gated.ejs

```diff
@@ -0,0 +1,29 @@
+<div class="resourcecenter-background half-padding-full">
+  <div class="container">
+    <div class="resourcecenter-fullWrapper resourcecenter-<%- type -%>">
+      <div class="<%- (type == 'video') ? 'col-sm-8 col-sm-push-2' : 'hidden-xs panel-flex col-md-4 col-sm-6' %> text-right">
+        <div class="resourcecenter-visualWrapper">
+        <% if(typeof image != 'undefined') { %>
+          <div class="resourcecenter-imageWrapper" >
+            <img class="resourcecenter-image" src="<%-image%>"/>
+          </div>
+        <% } %>
+        </div>
+      </div>
+      <div class="<%- (type == 'video') ? 'col-sm-8 col-sm-push-2' : 'col-md-8 col-sm-6 col-xs-12' %>">
+        <div class="resourcecenter-infoWrapper">
+          <div class="resourcecenter-titleFull"><h1><%- title %></h1></div>
```

Comment:
Instead of nested the H1 in a div can this just be:
```<h1 class="resourcecenter-titleFull"><%- title %></h1>```

Reply:
This is in all the new resource types. Do you want me to change it for all of them? I don't want them to be inconsistent. That would require a separate www change.

Reply:
ehh just leave it as-is for now then.